src/content/docs/cms/sanity/sanity-case-study.md

```diff
@@ -27,6 +27,59 @@ Over the years, the Sanity team has continued to improve and expand the platform
 **2018**: Continued refinement and expansion of Sanity CMS features to meet evolving user needs.
 **2019**: Further enhancements and improvements to Sanity CMS platform, focusing on developer-friendliness and customization.
 
+## About Sanity 
+
+### Increasing free resources in Sanity
+
+Sanity provides two APIs:
+
+1.	api.sanity.io: the live, uncached API. This is the default and will always give you the freshest data, but requests will be slower because they need to reach our back end on every request. Requests are also more costly because they will trigger more computation on our servers.
+2.	apicdn.sanity.io: the CDN-distributed, cached API, which is an opt-in feature that will give you very fast responses to requests that have been cached. We encourage most users to use this API for their front-ends if you expose your API to end users. For static builds you are better off using the live uncached API, to always get the latest version.
+To use the API CDN, simply use apicdn.sanity.io instead of api.sanity.io. Most clients provide a use CDN option that makes this switch seamless.
+
+
+Today we can use 20 frameworks, being React and Next.js the more popular.    
+You do not need to use php, Sanity uses Groq which is Sanity’s open-source query language.  It is possible to use graphQL if preferred. 
+When adding images to projects it’s possible to use a plug in, a powerful pipeline that  can take care of resizing, storage, deletion. 
```

Comment:
Here you can state what the plugin is call.

Reply:
Is it a plugin? isnt it just built into the studio?